libraries/streaming-e2e-tests/react-app/package.json

```diff
@@ -6,10 +6,11 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
+    "botframework-streaming": "next",
     "botframework-webchat": "4.10.1",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
-    "react-scripts": "3.4.3"
+    "react-scripts": "latest"
```

Comment:
Unsure if this is great because it affect reproducibility of test failures.

Reply:
We already do this in the ADO pipeline.

Reply:
Then why we are not using `react@latest` and `react-dom@latest` as well? And how about `testing-library/*@latest` and `botframework-webchat@latest`?

IMO, status quo, doesn't mean good/correct.

Reply:
That's a fair point, I'll pin it.